############          COMMON PARAMETERS        ############
planner:
  dt: 0.05                    # [s] this should be greater or equal to interpolation/sampling_dt
  fixed_yaw: false            # Fix the yaw to 0
  
interpolation:
  max_v: 18.0                 # [m/s]
  max_a: 10.0                # [m/s2]
  max_ang_v: 2.0             # [rad/s]
  max_ang_a: 2.0             # [rad/s2]
  sampling_dt: 0.05          # [s]